dataeng/resources/warehouse-transforms.sh

```diff
@@ -6,41 +6,54 @@ cd $WORKSPACE/warehouse-transforms
 # To install right version of dbt
 pip install -r requirements.txt
 
-
 cd $WORKSPACE/warehouse-transforms/projects/$DBT_PROJECT
 
-dbt clean --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
-dbt deps --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/ --profile $DBT_PROFILE --target $DBT_TARGET
+# Turn off automatic failure of this script if the command returns non-0
+set +e
 
-if [ "$SKIP_SEED" != 'true' ]
-then
-  dbt seed --full-refresh --profile $DBT_PROFILE --target $DBT_TARGET --profiles-dir $WORKSPACE/analytics-secure/warehouse-transforms/
-  if [ "$PUSH_ARTIFACTS_TO_SNOWFLAKE" = 'true' ]
+# Fails the job if a dbt command fails and uploads the dbt artifacts to Snowflake if the job is configured for it
+function postCommandChecks {
```

Comment:
Nit: in addition to the echo statement, a comment explaining the function arguments would be good.